bayesian/utils.py

```diff
@@ -31,8 +32,19 @@ def server_run_flow(flow_name, flow_args):
     Setup.connect_if_not_connected()
     return run_flow(flow_name, flow_args)
 
+def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
+    args = {
+        'external_request_id': uuid.uuid4().hex,
+        'data': {
+            'api_name': 'component_analyses',
```

Comment:
`server_create_component_bookkeeping` function is used only for bookkeeping component-analyses.